pkg/model/model.go

```diff
@@ -34,11 +35,13 @@ import (
 
 const modelZooDB = "sqlflow"
 const modelZooTable = "sqlflow.trained_models"
+const modelMetaFileName = "model_meta.json"
 
 // Model represent a trained model, which could be saved to a filesystem or sqlfs.
 type Model struct {
 	workDir     string // We don't expose and gob workDir; instead we tar it.
 	TrainSelect string // TrainSelect is gob-encoded during I/O.
+	Meta        []byte // Meta json object
```

Comment:
Why not keep a JSON object or `map[string]interface{}`?

Reply:
I searched some json lib, like [jsoniter](http://jsoniter.com/migrate-from-go-std.html), [jsonparser](https://github.com/buger/jsonparser), go std json, none of them provide JSON object. And it still not easy to use when we parse the json string to map[string]interface{}. I'm thinking about keeping the original byte array and use bellowing method to get anything we want. It is not the fastest, but may be easy to use. How do you think?
```go
jsoniter.Get(bytes, key).ToVal(OurStruct)  // parse a Struct
jsoniter.Get(bytes, key).String()
```

Reply:
Can you please try: https://blog.golang.org/json Decoding arbitrary data?

Reply:
I tried this, provide that parse json string to map[string]interface{} is possible, but, it not so easy to use the field, especially when the filed's path is deep. We can try `jsonparser` to save the meta as a Json struct. It is not a byte array and also keep the simplicity to get fields.